CHANGELOG.md

```diff
@@ -24,7 +24,15 @@ Adding a requirement of a major version of a dependency is breaking a contract.
 Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
-[Unreleased]: https://github.com/atlassian/jira-actions/compare/release-3.20.3...master
+[Unreleased]: https://github.com/atlassian/jira-actions/compare/release-3.20.4...master
+
+## [3.20.4]
+[3.20.4]: https://github.com/atlassian/jira-actions/compare/release-3.20.3...release-3.20.4
+
+### Fixed
+- Fix switching Activity Tabs on Jira 9.8.0. Fix [JPERF-1091].
```

Comment:
Discussed concern about verification of this fix.

@ewefie correctly suggested that we should add 9.8.0 to our test matrix.

Reply:
Good idea, let's add in in this PR.

Reply:
Old code fails: ![image](https://user-images.githubusercontent.com/1194704/236827766-8d04f2dd-a462-4512-8109-b1dc40b24c3f.png)
New code doesn't throw, but also doesn't scroll far enough:
[Screencast from 08.05.2023 14:48:40.webm](https://user-images.githubusercontent.com/1194704/236828594-f803d08f-dd2f-45d3-9d6f-80cc5ce1329f.webm)